SUPPORT.md

```diff
@@ -0,0 +1,38 @@
+## Kubernetes User Support
+
+Welcome to Kubernetes! We use GitHub for tracking bugs and feature requests. 
+This isn't the right place to get support for using Kubernetes, but the following 
+resources are available below, thanks for understanding.
+
+### Stack Exchange
+
+The Kubernetes Community is active on Stack Overflow, you can post your questions there: 
+
+* [Kubernetes on Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
+
+  * Here are some tips for [about how to ask good questions](http://stackoverflow.com/help/how-to-ask).
+  * Don't forget to check to see [what's on topic](http://stackoverflow.com/help/on-topic).
+
+### Real-time Chat
+
+* [Slack](https://kubernetes.slack.com) ([registration](http://slack.k8s.io)):
+The `#kubernetes-users` and `#kubernetes-novice` channels are usual places where 
+people offer support.
+
+* Also check out the 
+[Slack Archive](http://kubernetes.slackarchive.io/) of past conversations.
+
+### Mailing Lists/Groups
+
+* [Kubernetes-users group](https://groups.google.com/forum/#!forum/kubernetes-users)
+
+
+### Documentation 
+
+* [User Documentation](https://kubernetes.io/docs/) 
+* [Troubleshooting Guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/)
+
+
```

Comment:
Maybe we also want to redirect subcomponents issues to their repos in this SUPPORT file?

Such as the below info once existed in issue template.

 **Note:** Please file issues for subcomponents under the appropriate repo
 
  | Component | Repo                                                               |
  | --------- | ------------------------------------------------------------------ |
  | kubectl   | [kubernetes/kubectl](https://github.com/kubernetes/kubectl/issues/new) |
 | kubeadm   | [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm/issues/new) |